conans/client/installer.py

```diff
@@ -312,6 +318,24 @@ def _node_concurrently_installed(self, node, package_folder):
             if node.update_manifest == read_manifest:
                 return True
 
+    def _handle_node_editable(self, node):
+        # TODO: I need a valid 'package_folder', but I'm not going to use it
+        package_folder = self._client_cache.conan(node.conan_ref)
+        self._call_package_info(node.conanfile, package_folder=package_folder)
```

Comment:
Why do you need to call the package info method?

Reply:
It is necessary, to propagate the information to consumers.